README.md

```diff
@@ -1,2 +1,50 @@
+
 # uri-web
-Microservice version of CH URI service
+Microservice version of CH URI service.
+
+- Provides a URI style endpoint `/doc/company/<company number>` that returns basic company profile information, in a variety of formats, from the existing CHS APIs.   
+
+The format can be selected by specifying a suitable `Accept` header, or by adding a 'file type' extension, as follows:
+
+Format    |   Accept header(s)   |  Extension
+----------|----------------------|-------------
+HTML|None, `text/html`|`.html` e.g.`/doc/company/<company number>.html`
+JSON|`application/json`|`.json`
+RDF|`application/rdf+xml`|`.rdf`
+XML|`application/xml`, `application/x-xml`|`.xml`
+CSV|`text/csv`, `text/comma-separated-values`, `application/csv`, `application/excel`|`.csv`
+YAML|`application/yaml`, `application/x-yaml`, `text/yaml`, `text/x-yaml`|`.yaml`
+
+Please see https://www.gov.uk/guidance/companies-house-data-products#uri-info for further information on the data provided.
+
+This application is written using the [Spring Boot](http://projects.spring.io/spring-boot/) Java framework.
+
+### Requirements
+In order to build and run this Web App locally you will need to install:
+
+- [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- [Maven](https://maven.apache.org/download.cgi)
+
+### Getting Started
+
+1. [Configure your service](#configuration) if you want to override any of the defaults.
+1. Run `make dist`
+1. Run `java -jar target/uri-web-unversioned.jar`
+
+
+### Configuration
+Required environment variables:
+Key                | Description
+-------------------|------------------------------------
```

Comment:
Other READMEs have a separate column in the table for the example value. They also have a column for the default value if any.

Reply:
Thanks - I've split into separate columns